sentry/src/main/java/io/sentry/SentryClient.java

```diff
@@ -437,7 +434,11 @@ public void captureSession(final @NotNull Session session, final @Nullable Objec
 
     try {
       final SentryEnvelope envelope =
-          buildEnvelope(transaction, filterForTransaction(getAttachmentsFromScope(scope)));
+          buildEnvelope(
+              transaction,
+              filterForTransaction(getAttachmentsFromScope(scope)),
+              null,
+              scope != null ? TraceState.create(transaction, scope, options) : null);
```

Comment:
The only information extracted from scope is the user, which is an optional information.
So, why create the TraceState only if there is a scope?